common/ferrostar/src/routing_adapters/osrm/models.rs

```diff
@@ -162,6 +162,8 @@ pub struct BannerInstruction {
     pub distance_along_geometry: f64,
     pub primary: BannerContent,
     pub secondary: Option<BannerContent>,
+    // sub-maneuver information
+    pub sub: Option<SubManeuverBannerContent>,
```

Comment:
not sure if using `BannerContent` here is preferable or not, but many of these are different.